index.src.html

```diff
@@ -61,8 +61,8 @@
 ## Registration (<a>embedded authenticator</a> mode) ## {#registration-embedded}
 
 - On the phone:
-    * User goes to example.com in the browser, and signs in using whatever method they have been using (possibly a legacy
-        method such as a password).
+    * User goes to example.com in the browser and signs in to an existing account using whatever method they have been using
```

Comment:
I still believe "sign in to XXX" (no hyphen) is the correct usage.

Examples:

- "[Sign in to your account](https://login.microsoftonline.com/)" (see page title)
- "[Sign in to Gmail](https://support.google.com/mail/answer/8494?hl=en)"
- "[Sign in to iCloud.com](https://support.apple.com/kb/PH2610?locale=en_US)"

(Paypal seems to use "Log in" everywhere but they don't hyphenate either.)